tests/automation_framework/src/utilities/worker_utilities.py

```diff
@@ -212,28 +212,30 @@ def configure_data(action_obj, input_json, worker_obj, pre_test_response):
 def handle_value(val):
```

Comment:
The function is not being used. Delete it

Reply:
Fixed